src/main/java/com/bobocode/petros/annotation/Dependency.java

```diff
@@ -8,4 +8,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE, ElementType.METHOD})
 public @interface Dependency {
+    String name() default "";
```

Comment:
Without default value _**name**_ parameter will be mandatory. Dependency might not have a **_name_** parameter. In case of class dependency without **_name_** we'll use class name starting with a lowercase letter, and in case of configClass - it'll be a method name.